### DISCUSSION TITLE
gpg: no valid OpenPGP data found.

### DISCUSSION THREAD

**Discussion by Raghavek1** [jiangcuo/Proxmox-Arm64#42] [General]
Hi

Could you please help me to resolve the below issue ?

echo "deb https://mirrors.apqa.cn/proxmox/ pvearmdev main">/etc/apt/sources.list.d/foxi-devel.list
curl -L https://mirrors.apqa.cn/proxmox/gpg.key |apt-key add 
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0Warning: apt-key is deprecated. Manage keyring files in trusted.gpg.d instead (see apt-key(8)).
100   153  100   153    0     0    213      0 --:--:-- --:--:-- --:--:--   213
gpg: no valid OpenPGP data found.

Thanks
Raghav

**Comment by jiangcuo** [comment]
https://github.com/jiangcuo/Proxmox-Port/blob/main/help/repo.md
